apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/store.ts

```diff
@@ -21,12 +21,10 @@ export const selectors = {
 
 const STORE_KEY = 'automattic/starter-page-layouts';
 
-registerStore( STORE_KEY, {
-	// In reality the store can dispatch any action, however `reducer` has a
-	// strongly typed action type to make the typings inside the function
-	// easier to work with.
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	reducer: reducer as any,
-	actions,
-	selectors,
-} );
+export const store = createReduxStore( STORE_KEY, { reducer, actions, selectors } );
+
+// TODO: Why is this needed?
+if ( ! starterPageTemplatesConfig.didRegisterStore ) {
```

Comment:
I don't think that registered check is necessary:

https://github.com/WordPress/gutenberg/blob/1b6b4b8d8439b7cc0b0d90334a3b7cccaa665b45/packages/block-directory/src/store/index.js#L40

There are a bunch of core stores registered that same way, and I haven't noticed that practice there.

Reply:
Removing the check should also address the TS error that the `type_check_etk` CI task is failing with.